stdlib/public/runtime/SwiftRT-COFF.cpp

```diff
@@ -64,9 +65,9 @@ static void swift_image_constructor() {
   { reinterpret_cast<uintptr_t>(&__start_##name) + sizeof(__start_##name),     \
     reinterpret_cast<uintptr_t>(&__stop_##name) - reinterpret_cast<uintptr_t>(&__start_##name) - sizeof(__start_##name) }
 
-  sections = {
+  new (&sections) swift::MetadataSections {
```

Comment:
This is rather unfortunate.  The desire was to keep this something that could be statically emitted.  I suppose that the placement new allocator means that it is just a function call to the delegating ctor?

Reply:
It's syntactically required because `std::atomic` has a deleted copy constructor, which is annoying as heck-all, but it _should_ actually result in identical assembly as what was there previously.